usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction_count.md

```diff
@@ -17,11 +17,21 @@ Returns the counts of transaction records which match the keyword grouped by awa
 
     + Attributes (object)
         + `filters` (required, AdvancedFilterObject)
-        The `keywords` filter is mandatory for this endpoint
+            Must at least provide `award_type_codes` in filters object.
     + Body
 
 
-            {"filters":{"keywords":["test"]}}
+            {
+                "filters": {
+                    "keywords": ["test"],
+                    "award_type_codes": [
+                        "A",
+                        "B",
+                        "C",
+                        "D"
```

Comment:
Just double checking.. the award type codes are stored on the frontend here - https://github.com/fedspendingtransparency/usaspending-website/blob/master/src/js/dataMapping/award/descriptionsForAwardTypes.js .  Are there any changes to the award type codes with this change?

Reply:
No, they match the existing endpoint `spending_by_transaction`